x-pack/legacy/plugins/siem/public/components/embeddables/embedded_map_helpers.tsx

```diff
@@ -5,7 +5,9 @@
  */
 
 import uuid from 'uuid';
+import React from 'react';
 import { npStart } from 'ui/new_platform';
+import { OutPortal, PortalNode } from 'react-reverse-portal';
```

Comment:
Just noting (per a conversation with @spong ), that the re-parenting provided by `react-reverse-portal` here is required here because when `MapToolTip` is rendered in the embeddables component tree (the default behavior), it doesn't have access to the Redux store, which is required to register and un-register the draggable.

Reply:
That's correct. Adding the following comment to `embeddable_map` to make it clear for anyone meandering through this code at a later date. Thanks for specifying!

``` tsx
    // This portalNode provided by react-reverse-portal allows us re-parent the MapToolTip within our
    // own component tree instead of the embeddables (default). This is necessary to have access to
    // the Redux store, theme provider, etc, which is required to register and un-register the draggable
    // Search InPortal/OutPortal for implementation touch points
    const portalNode = React.useMemo(() => createPortalNode(), []);
```